Makefile

```diff
@@ -118,9 +121,10 @@ docker-buildx: ## Build and push docker image for the manager for cross-platform
 .PHONY: build-installer
 build-installer: manifests generate kustomize ## Generate a consolidated YAML with CRDs and deployment.
 	mkdir -p dist
-	echo "---" > dist/install.yaml  # Add a document separator before appending
-	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
-	$(KUSTOMIZE) build config/default >> dist/install.yaml
+	rm -rf config/release
+	cp -r config/manager config/release
+	cd config/release && $(KUSTOMIZE) edit set image controller=${IMG}
+	$(KUSTOMIZE) build config/release > ${INSTALLER_PATH}
```

Comment:
any do we need any of this ?

Reply:
We don't want to modify dist/install.yaml or config/default when we are using custom IMGs

Reply:
then how are we ever going to be able to change the dist manifests when we want ?

Reply:
and furthermore, the flow when custom images are built is aligned w/ deploy. This just makes sense for releases. 

Maybe I'm missing something.

Reply:
Right now current git porcelain checks if we modify `dist/install.go`